src/cli/mod.rs

```diff
@@ -40,3 +42,32 @@ impl CommandExecute for HarmonicCli {
         }
     }
 }
+
+pub(crate) async fn signal_channel() -> eyre::Result<(Sender<()>, Receiver<()>)> {
+    let (sender, reciever) = tokio::sync::broadcast::channel(100);
```

Comment:
Does this `100` have any particular meaning, or was it chosen by the fair dice roll of xkcd fame?

Reply:
Fair dice roll!